extension/background/youtubei.js

```diff
@@ -54,3 +60,105 @@ function parseTimestamp(ts) {
     const hours = parseInt(parts[2]) || 0
     return secs + (60 * mins) + (60 * 60 * hours)
 }
+
+// time-comments related code below
+
+const INNERTUBE_API_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8"
+const INNERTUBE_CLIENT_NAME = "WEB"
+
+async function fetchTimeComments(videoResponse) {
```

Comment:
`fetchTimeComments` function name is confusing because it seems like it returns comments but actually it returns chapters. We could use `fetchChaptersFromComments` or something similar.

Reply:
Okay, I'll rename the function as you suggest.